src/handbook/company/security/index.md

```diff
@@ -4,6 +4,15 @@ navTitle: Security
 
 # Security
 
+To ensure the safety and security of our company, employees and customers, FlowForge
```

Comment:
Usually I know I'm bikeshedding, but in this case it really needed an extra comma
```suggestion
To ensure the safety and security of our company, employees, and customers, FlowForge
```

Reply:
Heh - blame Vanta and their template's lack of Oxford commas.